liiweb/urls.py

```diff
@@ -15,6 +15,11 @@
         views.LegislationListView.as_view(variant="all"),
         name="legislation_list_all",
     ),
+    path(
+        "legislation/regulations",
```

Comment:
Let's be consistent with the internal name for this. "subsidiary legislation" or "subleg" for short is the technical term. There are multiple variants of that, such as "regulations", "SIs", etc.

So I think "legislation/subsidiary" is best and either "subsidiary" elsewhere or "subleg".

Reply:
Changed.